airbyte-cdk/bulk/toolkits/extract-jdbc/src/main/kotlin/io/airbyte/cdk/jdbc/SSLCertificateUtils.kt

```diff
@@ -0,0 +1,282 @@
+/*
```

Comment:
this file is copied from cdk1.0 

https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/java/airbyte-cdk/core/src/test/kotlin/io/airbyte/cdk/db/util/SSLCertificateUtilsTest.kt

Reply:
Furthermore, please update source-firetruck to use this code.